packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -17,7 +12,7 @@ const selectors = {
 		`${ panel } button.is-active:has-text("${ tabName }")`,
 
 	// General section-related
-	section: ( name: ArticleSections ) =>
+	section: ( name: string ) =>
```

Comment:
I chose to remove `ArticleSections` type because it proved to be too restrictive against an indeterminate and quickly changing set of sidebar panel sections. 

For instance, there is a new section that was previously not present when this type was defined - `Newsletter` - and instead of type-gating, I will leave the onus of ensuring the correct heading to the caller.